lametro/search_indexes.py

```diff
@@ -50,8 +51,18 @@ def prepare_attachment_text(self, obj):
         )
 
     def prepare_legislative_session(self, obj):
-        start_year = obj.legislative_session.identifier
-        end_year = int(start_year) + 1
+        if len(obj.actions_and_agendas) > 0:
```

Comment:
Let's store the result of `actions_and_agendas` under a variable so it is only evaluated once.